calcom/cal.com
# Setup Wizard Not Pulling Timezone

## What does this PR do?



Fixes #3654 

## Demo
https://www.loom.com/share/f9ae6d74a63e4a30bca2e4f44b0ae4ad


**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- Create a new account
- Navigate to the first step and check to see if your timezone pulls through or not

## Checklist



- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
